Jenkinsfile-Release

```diff
@@ -37,6 +37,12 @@ properties([
       name: 'chartVersion',
       description: '(Optional) Version of the Chart, like "139.0.0". If omitted, it will be calculated from the appVersion.',
     ),
+  ]),
+  pipelineTriggers([
+    triggers: [
+      upstream(upstreamProjects: 'insight/insight-brain/Release IQ Server',
+               threashold: hudson.model.Result.SUCCESS)
+    ]
```

Comment:
I would like this release build to be triggered once the Release IQ Pipeline build successfully finishes. However this build takes several params. Is this possible?
```
```
https://github.com/sonatype/helm3-charts/blob/9f51f621ed176074cc816a51c44819c17aa4d5ef/Jenkinsfile-Release#L26..L40

Reply:
> I would like this release build to be triggered once the Release IQ Pipeline build successfully finishes. However this build needs 2 parameters so I'm not sure if this is possible? 🤔

Pseudocode, something like this should work:
```
        def upstream = currentBuild.rawBuild.getCause(hudson.model.Cause$UpstreamCause)
        def parameters = upstream?.actions.find{ it instanceof ParametersAction }?.parameters
... do something with parameters
```

Reply:
> 

Thanks Kelly. That should work and also looks familiar.  I think we do the same when deploying IQ to staging / production. Will close this PR and look at the code.